src/login.tsx

```diff
@@ -57,13 +58,13 @@ export default function LoginPage() {
       id="login-page"
       data-testid='login'
       component="form"
+      noValidate
```

Comment:
What is this for?

Reply:
I can see these fields being used in some MUI examples, in their docs, which is why I had them. But I don't see any difference in behavior, so I removed the noValidate and I also changed it to autoComplete="on". Autocomplete isn't perfect, but it's a nice-to-have.